packages/resolver/lib/sources/truffle/index.ts

```diff
@@ -68,6 +73,30 @@ export class Truffle implements ResolverSource {
         this.options.compilers
       );
       return { body: addressSource, filePath: importPath };
+    } else if (importPath === `truffle${path.sep}Console.sol`) {
+      // calculating this in webpack env breaks
+      let unbundledGanacheConsoleSol;
+      // @ts-ignore
+      if (typeof BUNDLE_VERSION === "undefined") {
+        unbundledGanacheConsoleSol = path.resolve(
+          path.join(
+            require.resolve("@ganache/console.log"),
+            "..",
+            "..",
+            "console.sol"
+          )
```

Comment:
I think this should work:

```suggestion
          require.resolve("@ganache/console.log/console.sol")
```